### DISCUSSION TITLE
Create-strapi-app stuck in older version of Strapi

### DISCUSSION THREAD

**Discussion by jiv-e** [strapi/strapi#6839] [Legacy Topics]
**Describe the bug**
Create-strapi-app installs older than current Strapi version. 

**Steps to reproduce the behavior**

1. Install Strapi 3.0.0-beta.17.4 globally. npm install strapi@^3.0.0-beta.17.4 -g
2. Uninstall Strapi globally. npm uninstall strapi -g
3. Run: npx create-strapi-app my-project --quickstart
4. Check the version create-strapi-app installed.
5. Strapi version 3.0.0-beta.17.4 is installed.

**Expected behavior**
Current version of Strapi is installed.

**System**

- Node.js version: 12.14.1
- NPM version: 6.13.6
- Operating system: OSX

**Additional context**
I also tried to install Strapi 3.0.0-beta.18 globally instead of beta.17. Create-strapi-app is still stuck in beta.17. Is the Strapi version cached somewhere?

**Comment by jiv-e** [comment]
Maybe related: #6655

**Comment by derrickmehaffy** [comment]
This is not an issue with Strapi but the users environment. Moving to a discussion

**Comment by jiv-e** [comment]
Even installing the current version of Strapi doesn't help.

**Reply by derrickmehaffy** [reply]
`npx create-strapi-app@latest my-project --quickstart`

**Reply by derrickmehaffy** [reply]
But global packages do not relate to npx, they are two different tools
You can use `which strapi` command in a terminal and figure out where your global packages are stored and manually remove them

**Reply by jiv-e** [reply]
Yeah.. That is the strange thing I still don't understand. Where is the old version coming from. I first suspected global npm package as I couldn't figure out any other reason for npx using the older version. It's clear now that the version is coming somewhere else.

`npx create-strapi-app@latest` solved the problem, but after running `npx create-strapi-app` without the `@latest` still uses the 3.0.0-beta.17.4. Any ideas how to fix this?

**Reply by derrickmehaffy** [reply]
I have no idea, I'm not a mac user (Linux Mint) but using the `which` command will tell you were the package is coming from, it's usually in some bin dir that you can navigate a few up (it should be symlink to the actual global node_modules dir).

**Reply by derrickmehaffy** [reply]
From here: https://medium.com/better-programming/how-to-check-your-globally-installed-npm-packages-32a14469b95a

`npm list -g`

**Reply by derrickmehaffy** [reply]
In my case I don't use npm global and instead use yarn global

**Comment by jiv-e** [comment]
I figured it out. As you said this is not related to npm global packages. At first I was thinking it was.

For some reason I thought that `npx create-strapi-app` somehow downloads the newest script every time it's run. It just came to my mind to try `which create-strapi-app` and it pointed me here: `/Users/###/.config/yarn/global/node_modules/.bin/create-strapi-app`.

I don't know why it was inside a yarn config folder. I don't remember using yarn earlier... or maybe I did, not sure. I removed the yarn config folder, run the `npx create-strapi-app` command again and it worked. I didn't need to use `@latest` anymore. Strangely it didn't add the `.config/yarn` folder there anymore and `which create-strapi-app` returned: `create-strapi-app not found`.
I hope this helps if anyone has a similar problem.

Thanks @derrickmehaffy, I greatly appreciate your help!!

**Reply by derrickmehaffy** [reply]
No problem, glad you figured it out (and no npx doesn't always download and use the latest, hence why you normally want to specify that with `npx create-strapi-app@latest` )
